swagger/api.yml

```diff
@@ -121,22 +175,56 @@ paths:
                 example: ["btcr","sov","v1","key"]
 components:
   schemas:
+    Error:
+      description: Request parameters for the create operation.
+      type: object
+      properties:
+        errorCode:
+          type: string
+        errorMessage:
+          type: string
+
     CreateRequest:
       description: Request parameters for the create operation.
       type: object
       additionalProperties: false
+      required:
+        - did
+        - state
+        - didDocument
       properties:
-        jobId:
+        did:
+          description: DID we want to create/register
           type: string
+          format: W3C-compliant decentralized identifier
+          example: did:ion:EiClkZMDxPKqC9c-umQfTkR8vvZ9JPhl_xLDI9Nfk38w5w
+        state:
```

Comment:
State is returned to the client. 
Q: should we keep the name and change the "state" in the response or we keep the "jobId" and update the description?